.env.example

```diff
@@ -1,7 +1,56 @@
-DEBUG=
+# Set the TMPDIR to be used
+TMPDIR="$TMPDIR"
+
+# Accessing AWS for testnet.polykey.io and mainnet.polykey.io deployment
+AWS_DEFAULT_REGION='ap-southeast-2'
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+
+# Container name
+PK_AWS_ECS_CONTAINER='polykey'
+# Container registry domain
+PK_AWS_ECS_REGISTRY='015248367786.dkr.ecr.ap-southeast-2.amazonaws.com'
+# Container repository
+PK_AWS_ECS_REPOSITORY="$PK_AWS_ECS_REGISTRY/$PK_AWS_ECS_CONTAINER"
+# ECS cluster name
+PK_AWS_ECS_CLUSTER='polykey-testnet'
+# ECS service name
+PK_AWS_ECS_SERVICE='polykey-testnet'
+# ECS task definition name
+PK_AWS_ECS_TASK_DEFINITION='polykey-testnet'
+# ECS log group
+PK_AWS_ECS_LOG_GROUP="/ecs/$PK_AWS_ECS_TASK_DEFINITION"
```

Comment:
These all are not needed anymore, we're not using env variables to configure things, instead we will have IaC inside the `infra` section.